### PR TITLE
Laura Lyckholm Design Handoff Project W-11

### PR DESCRIPTION
**Pull request by LauraLyckholm**
See ReadMe and Comments for notes and thoughts.
View live here: https://align-studio-yoga.netlify.app/about-us 

UX-student: Linnea Pettersson
Design can be found here: https://www.figma.com/file/lcGwrY2phEf6Y1CyZK9ojt/Week-9---Designer-Developer-Handover-(Copy)?type=design&node-id=0%3A1&mode=design&t=bSLX9VnD6ISQ7pdV-1

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import { useTranslation } from "react-i18next";
+import ukFlagIcon from '/assets/icons/united-kingdom.png';
+
+// Component for the footer
+export const Footer = () => {
+    // Get the translation function from the i18next hook
+    const { t, i18n } = useTranslation();
+    // State to store the width of the window
+    const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+    // State to control whether the flag icon should be shown or not
```
Comment (IdahCollin) [inline src/components/Footer/Footer.jsx:14]: Good job using state here! I don't get the Swedish flag to show though.

File: src/components/Footer/SubscribeSection.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+export const SubscribeSection = () => {
+    const { t } = useTranslation();
+
+    // States for the form
+    const [submit, setSubmit] = useState(false); // Triggers useEffect
+    const [emailInput, setEmailInput] = useState(""); // Captures the email on input change
+    const [submittedEmail, setSubmittedEmail] = useState(""); // Captures the email on form submission
+    const [emailValid, setEmailValid] = useState(true); // Validates the email on input change
+
+    // Function to handle form submission
```
Comment (IdahCollin) [inline src/components/Footer/SubscribeSection.jsx:15]: Nice!

File: src/components/Footer/SubscribeSection.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    }, [submit, submittedEmail]);
+
+    // Function to handle input change and validation of emailaddress
+    const handleInputChange = (event) => {
+        const email = event.target.value;
+        setEmailInput(email); // Captures the email on input change
+        setEmailValid(validateEmail(email)); // Validates the email on input change, sets emailValid to true or false
+    }
+
+    // Function to validate email address
```
Comment (IdahCollin) [inline src/components/Footer/SubscribeSection.jsx:40]: Nice detail here Laura! Given the amount of time, this is really nicely done!

File: src/components/Header/BackArrow.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { useLocation, Link } from "react-router-dom";
```
Comment (IdahCollin) [inline src/components/Header/BackArrow.jsx:1]: Great way of solving this! Adding an error page is a bonus!

File: src/components/Header/ScrollToTop.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { useEffect } from "react";
```
Comment (IdahCollin) [inline src/components/Header/ScrollToTop.jsx:1]: Again, love your eye for details. Adding this makes the site so much better!

File: src/components/SubHeroComponent/SubHeroComponent.jsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { Button } from "../Links/Button";
```
Comment (IdahCollin) [inline src/components/SubHeroComponent/SubHeroComponent.jsx:1]: Good job using re-usable components!

File: src/index.css [hunk 7]
```diff
@@ -1,13 +1,186 @@
-:root {
+/* ------------ Font Imports ------------ */
```
Comment (IdahCollin) [inline src/index.css:1]: I haven't used Tailwind yet so this was nice to see to learn! Great with clear comments

File: src/pages/LandingPage.jsx [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+                textImageHeading={t("landingPage.virtualClasses.h2")}
+                textImageParagraph1={t("landingPage.virtualClasses.virtualClassesParagraph1")}
+                textImageParagraph2={t("landingPage.virtualClasses.virtualClassesParagraph2")}
+                className={"brown-to-green thick"}
+                textImageButton={t("landingPage.virtualClasses.virtualClassesButton")} />
+        </>
+    )
+}
+
+// imageOnLeft = true, // imageOnLeft is a prop that defaults to true
```
Comment (IdahCollin) [inline src/pages/LandingPage.jsx:42]: Did you leave this here on purpose? Always nice to keep in learning purpose if that's the case!

**Review by IdahCollin — COMMENTED** [review]
The code works, fulfills requirements, follows guidelines and is well described! As it has a lot of components maybe a short explanation in read-me could be added next time? 

Given the amount of time we've had on this project, you should be really really proud of yourself Laura! You've added details as the "scroll-to-top", tried out different libraries/ways of styling, added pages even if it wasn't required and gave life to the UX-student's design. As always, Laurawesome, you're Awesome. Don't forget to pat yourself on the back! ❤️

File: README.md [hunk 9]
```diff
@@ -4,38 +4,31 @@
...
 
-## Instructions
+I implemented a lot of code to make my subscribe section work as intended. And then when I went over to the contact form I decided to use React Form Hook instead, which made the code shorter. I watched videos on it, and read the docs. This project is probably one of the few where I've used Google and Youtube basically as much as ChatGpt to find the correct information, instructions and tips and tricks. We talked a lot in our team as well, to solve issues. 
 
-<a href="instructions.md">
-   See instructions of this project
-  </a>
+## View it live
+
+https://align-studio-yoga.netlify.app/about-us
```
[the remark this answers is not in the document]
Comment (LauraLyckholm) [reply README.md:34]: Hahaha, whoopsi. Thanks for pointing that out!

**Pull request closed without merging**